PromiseTests/PromiseAllTests.swift

```diff
@@ -11,7 +11,7 @@ import XCTest
 
 class PromiseAllTests: XCTestCase {
     func testAll() {
-        weak var expectation = expectationWithDescription("`Promise.all` should wait until multiple promises are fulfilled before returning.")
+        weak var expectation = self.expectation(description: "`Promise.all` should wait until multiple promises are fulfilled before returning.")
```

Comment:
> This is because of the Swift 3 naming changes, so expectation is now a method name, as well as the local variable name.

oh, that's annoying. nothing to be done about it i guess